src/main/java/uk/gov/pay/card/db/loader/WorldpayPrepaidParser.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.pay.card.db.loader;
+
+import uk.gov.pay.card.model.PrepaidStatus;
+
+public class WorldpayPrepaidParser {
+
+    public static PrepaidStatus parse(String value) {
+        if ("Y".equals(value)) {
+            return PrepaidStatus.PREPAID;
+        }
+
+        if ("N".equals(value)) {
```

Comment:
I am wondering if you use `else if` will it be more "standard"? For this case the second if will not be executed, because you `return` from the method, but just wondering if `else if` makes sense

Reply:
Thanks Dan. I feel having the `return` statements means it doesn't require the `else if` and the function performs as expected whilst being simple to read.